### PR TITLE
Pepper 920 os2 translations

### PR DESCRIPTION
**Pull request by ssettipalli**
Create a updated english translation file with ALL Osteo1 and Osteo2 various version activities.
Intent is to get spanish translations for the text

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/osteo/activities/prequal.conf [hunk 1]
```diff
@@ -34,10 +34,8 @@
                 {
                   "name": "prompt_PREQUAL_SELF_DESCRIBE",
                   "translations": [
-                    {
-                      "language": "en",
-                      "text": "First, who is signing up for the Osteosarcoma Project? <br><em>Check all that apply</em>"
-                    }
+                    { "language": "en", "text": ${i18n.en.prequal.describe_prompt} },
+                    { "language": "es", "text": ${i18n.es.prequal.describe_prompt} }
```
Comment (pegahtah) [inline pepper-apis/studybuilder-cli/studies/osteo/activities/prequal.conf:38]: Shouldn't there be an i18n/es.conf in this PR as well?

Comment (ssettipalli) [reply pepper-apis/studybuilder-cli/studies/osteo/activities/prequal.conf:38]: es.conf exists and once we get new es translations from CMI, we will updates es.conf

Comment (ssettipalli) [reply pepper-apis/studybuilder-cli/studies/osteo/activities/prequal.conf:38]: No changes to es.conf until we get translations from cmi

Comment (pegahtah) [reply pepper-apis/studybuilder-cli/studies/osteo/activities/prequal.conf:38]: I see!!

**Review by pegahtah — APPROVED** [review]
TY!

**Pull request merged**